### ISSUE TITLE
Intelligence Report: IADS network

### ISSUE THREAD

**Issue by 132ndNeck** [132nd-vWing/OPUJ_development#10]
Create a Intelligence report for a IADS network.
The IADS network must have a Command Center, EW radars, SAM sites, communication nodes and powersupply.

Easy explanation of the concept availeble here:
https://github.com/walder/Skynet-IADS

Each military district will have their own IADS network.

Will be made into a scripting issue/task once we know how the IADS network should be setup.
Once we have the layout, then it will also added to target production, so that the network can be attack by destroying commandcenter, powersupply, communication nodes or EW.

**Comment by abeardedbear** [comment]
I'm finally pushing forward this document, after the gathering of information. 
We are focussing just on the SAM network, not on the anti-air support units for motorised and mechanised forces, right?

**Comment by 132ndNeck** [comment]
Correct. Focus is on the IADS network, not SAMs that support the ground forces.
